statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1670,27 +1670,42 @@ def llnull(self):
                                    freq_weights=self._freq_weights,
                                    scale=self.scale)
 
-    @cached_value
-    def llf(self):
-        """
-        Value of the loglikelihood function evalued at params.
-        See statsmodels.families.family for distribution-specific
-        loglikelihoods.
-        """
+    # Return the log-likelihood at the given scale, using the
+    # estimated scale if the provided scale is None.  In the Gaussian
+    # case with linear link, the concentrated log-likelihood is
+    # returned.
+    def _llf(self, scale=None):
```

Comment:
I think we could make this a public method `llf_scaled`
That would also be useful for a quasi-likelihood LR test with excess dispersion.